src/app/common/theme-provider.tsx

```diff
@@ -48,10 +50,26 @@ function setUserSelectedTheme(theme: UserSelectedTheme) {
 interface ThemeSwitcherProviderProps {
   children: React.JSX.Element | React.JSX.Element[];
 }
+
+function removeDefaultBg() {
```

Comment:
If we set the BG color in `bases.css` do we then need to remove it? We can have another BG colour on top of it then in the react components? 🤔